static/modal_style.css

```diff
@@ -0,0 +1,224 @@
+/* Modals */
+
+.modal-1 {
+  width: 100%;
+  height: 850px;
+  background-color: #3c3a39;
+  color: #fff5e1;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  font-family: 'Roboto', sans-serif;
+  border: 0;
+}
```

Comment:
- [ ] Good job implemeting the modal as requested, jhowever, your design does not match the given design in mobile view.Your bottom buttons are not spaced as in the given design and the colors for the buttonsdoes not match the figma design. Kindly fix this as it is part of the requirements

screenshot of requirements
![image](https://github.com/wacoo/Portfolio-Project/assets/81932478/b52d6c5a-3116-4318-97c2-15550eebe0df)

Given template
![image](https://github.com/wacoo/Portfolio-Project/assets/81932478/47d71615-2532-4b09-aaa9-13fb4e20f5e6)

Your design
![image](https://github.com/wacoo/Portfolio-Project/assets/81932478/82d0fccb-f72d-4fd8-861b-ad0422d03aa7)